doc/api_assets/api.js

```diff
@@ -136,6 +136,25 @@
     updateHashes();
   }
 
+  function setupFlavorSelectors() {
+    const kFlavorPreference = 'customFlavor';
+    const flavorSetting = sessionStorage.getItem(kFlavorPreference);
```

Comment:
I do not see any value in using a sessionStorage here. As the lifecycle would be contained to the current session. A localStorage makes more sense here.

Reply:
Pushed a new version with localStorage. This leaves some "garbage" values there though since it doesn't get removed, which may be more of an issue with localStorage than with sessionStorage. Adding a "removeItem" when the value is not "true" could fix that and keep it clean, but maybe not a big issue.

Reply:
I assume that localStorage should sync with the user preference. Either true or false :)

Reply:
Yes, but it's not really necessary to store the string "false" it can just be undefined aka be removed (removeItem) to avoid wasting localStorage space. But if it's fine for you, then it's fine for me, may just be nitpicking.

Reply:
Yeah, sorry, what I meant is that we should track both states. And for me the absence of a value is more than enough.

But we should remove then the value when set to false.

Reply:
Ok, pushed a fix to remove value when false.